lib/commands/general.js

```diff
@@ -1,6 +1,8 @@
-import { youiEngineDriverReturnValues } from '../utils';
+import { youiEngineDriverReturnValues, youiEngineKeyCode } from '../utils';
+import log from '../logger';
 
-let commands = {};
+
+let commands = {}, extensions = {};
```

Comment:
General is also a vague filename. What is this doing?

Reply:
following structure from another driver:
https://github.com/appium/appium-android-driver/blob/master/lib/commands/general.js#L366
but I could create a mobile.js file for these